tools/regression_test.py

```diff
@@ -430,6 +430,10 @@ def get_fps_metric(shell_res: int, pytorch_metric: dict, metric_info: dict,
         output_result[metric_name] = '-'
         if metric_key in backend_results:
             backend_value = backend_results[metric_key] * multi_value
+            if (metric_info[metric_name].get('task_name',
```

Comment:
could add `multi_value: 100` in yaml, just like mmdet.
https://github.com/open-mmlab/mmdeploy/blob/aee22f05d036de00d954286fcd36c73ba0609ea2/tests/regression/mmdet.yml#L11